### PR TITLE
Initial generic crv ERC20 zap

### PR DESCRIPTION
**Pull request by NicholasDotSol**
A generic erc20 zap contract for curve

### CODE REVIEW

**Review by jrhea — COMMENTED** [review]
minor nit in natspec comment

File: contracts/zaps/ZapCrvERC20.sol [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    /// @param __trancheBytecodeHash Hash of the Tranche bytecode.
+    /// @param _token1 Address of the first token.
+    /// @param _token2 Address of the second token.
+    /// @param _stableSwap Address of the curve pool.
+    /// @param _lp curve lp token.
+    constructor(
+        address __trancheFactory,
+        bytes32 __trancheBytecodeHash,
+        IERC20 _token1,
+        IERc20 _token2,
```
Comment (jrhea) [inline contracts/zaps/ZapCrvERC20.sol:86]:
```suggestion
        IERC20 _token2,
```

File: contracts/zaps/ZapCrvERC20.sol [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+        uint256 _ptExpected
+    ) external payable reentrancyGuard notFrozen returns (uint256, uint256) {
+        require(_amount != 0, "0 amount");
+
+        ITranche tranche = _deriveTranche(address(_position), _expiration);
+
+        //add liquidity with no minimum mint
+        stableSwap.add_liquidity([_amount, 0], 0);
+
+        uint256 outAmount = want.balanceOf(address(this));
```
Comment (jrhea) [inline contracts/zaps/ZapCrvERC20.sol:164]: I don't see `want` declared anywhere.  Should this be a contract level immutable var?

Comment (sentilesdal) [reply contracts/zaps/ZapCrvERC20.sol:164]: I also find the declaration of `want` to be wanting.

**Comment by NicholasDotSol** [comment]
replaced by #209

**Pull request closed without merging**